### PR TITLE
Improving docs and UI message

### PR DESCRIPTION
**Pull request by tobyash86**

### CODE REVIEW

File: dist/messages/messages.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
     "wrk_dir_label": "Working directory: ",
     "cmd_label": "Command line: ",
     "cmd_cannot_be_empty": "Command line cannot be empty.",
-    "failed_run_sa": "Run has been failed due to static analysis violations reported",
-    "failed_run_ut": "Run has been failed due to test failures reported",
-    "failed_run_non_zero": "Run has been failed due to non-zero dotTEST exit code: ",
+    "failed_run_sa": "dotTEST run failed because static analysis violations were reported",
+    "failed_run_ut": "dotTEST run failed because failed unit tests were reported",
+    "failed_run_non_zero": "dotTEST run failed with a non-zero dotTEST exit code:",
```
Comment (tobyash86) [inline dist/messages/messages.json:9]: Please keep white space characters. This line is missing space after colon

File: README.md [hunk 2]
```diff
@@ -69,24 +69,49 @@ jobs:
...
+To upload the SARIF report to GitHub, modify your workflow to add the `upload-sarif` action, which is part of the [github/codeql-action](https://github.com/github/codeql-action) repository).
+
+**Important!** To automatically upload the results, ensure that the path to the SARIF file is configured as the `${{ steps.[dottest_action_id].outputs.report }}` variable. In the following example the id of the dotTEST action is `dottest`.
 
 ```yaml
-    - name: Upload results to GitHub
+# Run Parasoft dotTEST Analysis and generate .sarif report.
+- name: Run dotTEST analyzer
+  id: dottest
+# Optional parameters
```
Comment (tobyash86) [inline README.md:80]: `uses` property is also mandatory. I'm thinking about removing "Optional parameters" comment and leave only the second line

**Review by tobyash86 — COMMENTED** [review]
Please reolve the following

File: README.md (hunk 2 above)
Comment (annastuchlik) [reply README.md:80]: I've added the "uses" property and removed "optional parameters".

Comment (tobyash86) [reply README.md:81]: Looks good

File: dist/messages/messages.json (hunk 1 above)
Comment (annastuchlik) [reply dist/messages/messages.json]: Fixed.

Comment (tobyash86) [reply dist/messages/messages.json]: Thanks!

**Pull request merged**